resources/jira/jnra/src/jnra/jiraSDK.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
+
+from jira import JIRA
+from jira.resources import Issue, Board, Sprint
+from jira.client import ResultList
+
+
+class JiraSDK():
+
+    PAGE_SIZE = 50
+
+    def __init__(self, api_key: str, api_usr: str) -> None:
+        self._jira = JIRA("https://ultimaker.atlassian.net/", basic_auth=(api_usr, api_key))
```

Comment:
Code style: Spaces around binary operators.
```suggestion
        self._jira = JIRA("https://ultimaker.atlassian.net/", basic_auth = (api_usr, api_key))
```

Similar issues exist in the code below as well.

Reply:
This is not a binary operator.